Code/Engine/Audio/AudioMetadataUtils.cpp

```diff
@@ -36,6 +42,41 @@ std::string GetFileExtension(const std::string& fileName)
 }
 
 //-----------------------------------------------------------------------------------
+bool IncrementPlaycount(const std::string& fileName)
+{
+    static const char* PlaycountFrameId = "PCNT";
+
+    if (fileName.find("FLAC") != std::string::npos || fileName.find("flac") != std::string::npos)
```

Comment:
For the future, we'd want to make this a case-insensitive comparison. This could be done by taking the file name and converting it to a lowercase string.